src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -6663,24 +6662,23 @@ public partial interface INotifyCompletion
     {
         void OnCompleted(System.Action continuation);
     }
-    [System.AttributeUsageAttribute((System.AttributeTargets)(1), AllowMultiple=true, Inherited=false)]
+    [System.AttributeUsageAttribute(System.AttributeTargets.Assembly, AllowMultiple=true, Inherited=false)]
     public sealed partial class InternalsVisibleToAttribute : System.Attribute
     {
         public InternalsVisibleToAttribute(string assemblyName) { }
         public bool AllInternalsVisible { get { throw null; } set { } }
         public string AssemblyName { get { throw null; } }
     }
-    [System.AttributeUsageAttribute((System.AttributeTargets)(8))]
     [System.ComponentModel.EditorBrowsableAttribute((System.ComponentModel.EditorBrowsableState)(1))]
+    [System.AttributeUsageAttribute(System.AttributeTargets.Struct)]
     public sealed partial class IsByRefLikeAttribute : System.Attribute
     {
         public IsByRefLikeAttribute() { }
     }
     public static partial class IsConst
     {
     }
-    [System.AttributeUsageAttribute((System.AttributeTargets)(32767), Inherited=false)]
-    [System.ComponentModel.EditorBrowsableAttribute((System.ComponentModel.EditorBrowsableState)(1))]
```

Comment:
Why is the EditorBrowsableAttribute being removed?

Reply:
Yep, missed that one. Had to undo most of the attribute changes.

Reply:
@weshaggard, it would be great if we didn't have to undo these changes every time. Do you know why the tool keeps skipping these certain attributes (`EditorBrowsableStateNever` and `Obsolete`)?

Reply:
It is most likely because they don't exist in the implementation. They were most likely only added to the ref and so when you regenerate from the implementation they get removed.

Reply:
Ah this looks to have been caused by @ericstj's PR https://github.com/dotnet/corefx/pull/32715 where he added these attributes to the exclude list https://github.com/dotnet/corefx/blame/master/eng/DefaultGenApiDocIds.txt#L57. @ericstj  did you intend for these to be removed?